### PR TITLE
Checkout form verification

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: ninjaStoreURL.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## NINJA STORE URL
+login = http://tutorialsninja.com/demo/index.php?route=account/login
+home = http://tutorialsninja.com/demo/index.php?route=common/home
+cart = http://tutorialsninja.com/demo/index.php?route=checkout/cart
```
Comment (Cxristian) [inline ninjaStoreURL.properties]: a blank space is missing

File: src/main/resources/log4j.properties [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+log4j.appender.FileAppender.File=log/log.log
+log4j.appender.HTML.File=./log/application.html
+
+# Here we define the layout and pattern
+log4j.appender.FileAppender.DatePattern='.'MM-dd-yyyy
+log4j.appender.FileAppender.layout=org.apache.log4j.PatternLayout
+log4j.appender.FileAppender.layout.ConversionPattern= %d{HH:mm:ss} %-5p [%C{1}][%M][%L] %m%n
+log4j.appender.HTML.layout=org.apache.log4j.HTMLLayout
+log4j.appender.HTML.layout.Title=Application log
+log4j.appender.HTML.layout.LocationInfo=true
```
Comment (Cxristian) [inline src/main/resources/log4j.properties]: a blank space is missing

File: src/test/java/steps/BuySteps.java [hunk 3]
```diff
@@ -61,11 +62,55 @@ public void assertAlertText() {
...
+    @When("the user goes to cart page")
+    public void theUserGoesToCartPage() {
+        PageTransporter.goToPage("cart");
+        cartPage = new CartPage();
+    }
+
+    /**
+     * Checkouts from cart page by pressing the button.
+     */
+    @And("the user checkouts")
```
Comment (Cxristian) [inline src/test/java/steps/BuySteps.java]: there doesn't have to be a sentence and

File: src/test/java/steps/BuySteps.java [hunk 4]
```diff
@@ -61,11 +62,55 @@ public void assertAlertText() {
...
+    @Then("the checkout options form is shown")
+    public void verifiesCheckoutOptionsSubtitle() {
+        Assert.assertEquals(checkoutPage.getFirstSubtitleText(), "Step 1: Checkout Options");
+    }
+
+    /**
+     * Adds a MacBook to shopping cart as prerequisite.
+     */
+    @Given("there is a product on shopping cart")
+    public void thereIsAProductOnShoppingCart() {
```
Comment (Cxristian) [inline src/test/java/steps/BuySteps.java]: you have to change the names of your methods

File: ninjaStoreURL.properties (hunk 1 above)
Comment (MelissaRoman06) [reply ninjaStoreURL.properties]: done

File: src/main/resources/log4j.properties (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/resources/log4j.properties]: done

File: src/test/java/steps/BuySteps.java (hunk 3 above)
Comment (MelissaRoman06) [reply src/test/java/steps/BuySteps.java]: changed to When

File: src/test/java/steps/BuySteps.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/test/java/steps/BuySteps.java]: done

**Review by Cxristian — APPROVED** [review]

File: src/test/resources/features/addressBook.feature [hunk 5]
```diff
@@ -1,7 +1,7 @@
 Feature: Address book management
 
   @CheckLogin
-  Scenario: User create a curriculum with the minimum data required
+  Scenario: Add new address to address book
     When the user goes to add address page
     And the user adds a new address with minimum data required
       | First Name   | TestName       |
```
Comment (alszla) [inline src/test/resources/features/addressBook.feature]: I understand this scenario is still in progress and it will eventually have an assertion, right?

**Review by alszla — APPROVED** [review]

**Pull request merged**